EIPS/eip-3085.md

```diff
@@ -76,8 +89,10 @@ If a field does not meet the requirements of this specification and the wallet d
 - `iconUrls`
   - If provided, **MUST** specify one or more URLs pointing to reasonably sized images that can be used to visually identify the chain.
 - `nativeCurrency`
-  - If provided, **MUST** describe the native currency of the chain using the `name`, `symbol`, and `decimals` fields.
-    - `decimals` **MUST** be a non-negative integer.
+  - If provided, **MUST** describe the native currency of the chain using the `name` and `symbol` fields, and `decimals` or `scalingFactor` fields.
+    - Either `decimals` or `scalingFactor` **MUST** be provided.
+      - If provided, `decimals` **MUST** be a non-negative integer.
+      - If provided, `scalingFactor` **MUST** be the `0x`-prefixed, hexadecimal string representation of a non-negative integer.
```

Comment:
Recommend asserting that `decimals` and `scalingFactor` are mutually exclusive and it is an error to include them both.